studio/schemas/deskStructure.ts

```diff
@@ -14,7 +14,7 @@ import { soMeLinksID } from "./documents/socialMediaProfiles";
 import { siteSettingsID } from "./documents/siteSettings";
```

Comment:
ok, we need to add common linting! 💡

Reply:
This is actually me fixing my own formatting from earlier, so I think we have the same now (using Prettier support in WebStorm)